FogCreek/Glitch-Community
# Profuse tea: Several styling fixes and a hopeful ssr fix

## Links
https://profuse-tea.glitch.me/
https://app.clubhouse.io/glitch/story/5366/incorrect-button-style-for-unfeaturing-project
https://app.clubhouse.io/glitch/story/5269/user-avatars-are-stretched-in-add-team-member-pop
https://app.clubhouse.io/glitch/story/6051/the-privacy-toggle-is-off-center
https://app.clubhouse.io/glitch/story/4531/typeerror-resetstate-is-not-a-function

## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/8932174/71213591-e832f280-2281-11ea-82e7-faa9fec7c54a.png)
![image](https://user-images.githubusercontent.com/8932174/71213635-0567c100-2282-11ea-807c-89d2f6fc9a0c.png)
![image](https://user-images.githubusercontent.com/8932174/71213645-10225600-2282-11ea-965b-0f460bd8991f.png)

## Changes:
- Converted the add note and unfeature buttons to size=small variant=secondary
- Fixed alignment of the bookmark button and dropdown for featured projects
- The featured project text wraps instead of overlapping the bookmark button
- User avatars don't get squished in the add user to team pop
- Centered the privacy toggle icon within the button
- Added a hopeful fix for no resetState, if the code fails to load it will try again on the next render rather than staying broken

## How To Test:
- Look at all of the things that got their styling fixed and make sure they look good
- Look at a private project item to make sure the non button private icon isn't broken
